RELEASE_PROCESS.md

```diff
@@ -0,0 +1,29 @@
+# Terraform Module Release Process
+
+## Steps
+
+1. Update versions in ./scripts/module_version.sh file for the old version and new version you want to release.
+
+For example to release version `v0.2.0`, the file should look like:
+
+```
+#!/bin/bash
+
+old="0\.1\.0"
+new=0.2.0
+
+...
+```
+
+2. Run `make generate_templates` from root level of repository. This should
+   update all examples and templates with the newest version.
+
+3. Submit a PR and get it reviewed and merged.
+
+4. Tag and push the new version to Github.
```

Comment:
I know we haven't done it lately repo, but I kind of like Github releases a bit. They'll make release notes and stuff which are cool for multiple PRs in the latest set of changes